pkg/agent/windows/remotedialer/client_windows.go

```diff
@@ -57,7 +57,7 @@ func connectToProxyWhileWindows(rootContext context.Context, proxyURL string, he
 	}
 
 	eg.Go(func() error {
-		connectRetryLimit := 10
+		connectRetryLimit := 20
```

Comment:
Why do you need ot increase the limit?

Reply:
Creating an `overlay` HNSNetwork will result in a longer network interruption than creating a `bridge` HNSNetwork, which typically takes almost 20-40s. It is possible that different machines will have different situations.

Reply:
Rollbacking to 10 times is ok because the agent will reconnect after the retring timeout, but with 20 times, the agent will not output reconnecting error logs easily when bootstrapping.

Reply:
Instead of increasing the limit, I will increase the sleep time. Can we use ticker and timeout context here?

Reply:
I agree to increase sleep time.